extensions/wikia/AdEngine/js/video/videoSettings.js

```diff
@@ -30,6 +34,25 @@ define('ext.wikia.adEngine.video.videoSettings', [
 			return Boolean(defaultStateAutoPlay || resolvedStateAutoPlay);
 		}
 
+		function isMoatTrackingEnabled(params) {
```

Comment:
We have a public method with the same name in this module. Is it possible to change name one of those?